src/main/java/nl/dtls/fairdatapoint/config/TemplateConfig.java

```diff
@@ -109,7 +109,7 @@ private static String literal(Literal literal, Options options) {
      */
     private static String simplifyUrl(IRI iri, Options options) {
         if (iri != null) {
-            String[] parts = iri.toString().split("/");
+            String[] parts = iri.getLocalName().split("[/#:]");
```

Comment:
`.getLocalName()` already returns the last part of the `IRI`, so it can replace the custom logic here (lines 112-115). See the [`IRI` implementation](https://github.com/eclipse/rdf4j/blob/master/core/model/src/main/java/org/eclipse/rdf4j/model/impl/SimpleIRI.java#L101-L108) and [the utility it is using](https://github.com/eclipse/rdf4j/blob/master/core/model/src/main/java/org/eclipse/rdf4j/model/util/URIUtil.java#L66-L82).

Reply:
Oh, ok I see. It still works because if there is no `#`, `/` or `:`, there is only one item in `parts` which is also the last one.